style.css

```diff
@@ -1,455 +1,1107 @@
+    padding-left: 2%;
+    margin-left: 10%;
+    width: 100%;
+    margin-top: 5rem;
+  }
+
+  .card-tittle {
+    margin-top: -15rem;
+    margin-left: -5.5rem;
+  }
+
+  .frame2-img {
+    margin-top: -1rem;
+    margin-left: -5.7rem;
+  }
+
+  .description {
+    margin-top: -0.25rem;
+    width: 85%;
+    margin-left: -5.3rem;
+  }
+
+  .languages-list {
+    margin-left: -5.3rem;
+    margin-top: -8.5rem;
+  }
+
+  .see-button {
+    margin-left: -5.4rem;
+  }
+
+  .order-2 {
+    order: 2;
+  }
+
+  /* About Section Desktop */
+  .about {
+    display: grid;
+    flex-direction: row;
+    width: 100%;
+    height: 90vh;
+  }
+
+  .left-grid {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    margin-left: -2rem;
+  }
+
+  .about-h1 {
+    display: flex;
+    white-space: nowrap;
+    margin-left: 10%;
+    width: 100%;
+    margin-top: -2.9rem;
+  }
+
+  .info-summary {
+    margin-left: 10%;
+    width: 27.2rem;
+    height: 102px;
+    margin-top: -2rem;
+  }
+
+  .connect2 {
+    margin-left: 10%;
+    margin-top: 0.5rem;
+  }
+
+  #social-icons2 {
+    margin-left: 7%;
+  }
+
+  .see-button2 {
+    margin-left: 10%;
+  }
+
+  .skill-set {
+    display: flex;
+    flex-direction: column;
+    width: 50%;
+    float: right;
+    margin-left: 35rem;
+    margin-top: -65rem;
+  }
+
+  .down-arrow {
+    margin-left: 41rem;
+  }
+
+  .language-list-container {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .language-list {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    padding-bottom: 3rem;
+  }
+
+  .js2 {
+    display: flex;
+    flex-direction: column;
+    width: 7.5rem;
+    height: 8rem;
+  }
+
+  .js-logo {
+    height: 4.5rem;
+    width: 4.4rem;
+  }
+
+  .js-text {
+    align-self: center;
+    margin-top: 5.6rem;
+    margin-left: -1.5rem;
+  }
+
+  .html2 {
+    display: flex;
+    flex-direction: column;
+    margin-top: -0.1rem;
+    width: 7.5rem;
+    height: 8rem;
+  }
+
+  .html-logo {
+    height: 4.5rem;
+    width: 4.4rem;
+  }
+
+  .html-text {
+    align-self: center;
+    margin-top: 5.8rem;
+    margin-left: -2.69rem;
+  }
+
+  .css2 {
+    display: flex;
+    flex-direction: column;
+    margin-top: 0.021rem;
+    width: 7.5rem;
+    height: 8rem;
+  }
+
+  .css-logo {
+    height: 4.5rem;
+    width: 4.4rem;
+  }
+
+  .css-text {
+    align-self: center;
+    margin-top: 5.5rem;
+    margin-left: -3.4rem;
+  }
+
+  .frameworks-text {
+    margin-left: 46.2rem;
+    margin-top: 6.2rem;
+  }
+
+  .right-arrow {
+    margin-left: 4rem;
+    margin-top: 6rem;
+  }
+
+  .separator-bottom {
+    margin-left: -28.5rem;
+  }
+
+  .skills-text {
+    margin-left: 46.2rem;
+    margin-top: 9.7rem;
+  }
+
+  .right-arrow2 {
+    margin-left: -5rem;
+    margin-top: 9.8rem;
+  }
+
+  .separator-bottom2 {
+    margin-left: -37.3rem;
+    margin-top: -0.5rem;
+  }
+
+  /* Contact Section Desktop */
+
+  .contact-section {
```

Comment:
- [ ] Kindly adhere to the project requirement because as I can see you implemented a few changes as per the previous code reviewer's request 👍🏻 but not complete your design has too much gap or spacing at the right-hand-side at `=768px` screen size, which makes your design `inconsistent` and looks unprofessional. Please have a look at the below-attached image and table.

### Requirement
![image](https://user-images.githubusercontent.com/66476329/236188683-4224cc77-237d-413d-9942-91297c87687b.png)

### Design Comparison
<table>
<tr>
<th>Figma Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

![template_1_contact-section](https://github.com/Edgahkipkemoi/mobile/assets/66476329/dd8ccde1-7b73-434f-9a0e-995b6f970305)

</td>
<td>

![Screenshot from 2023-06-17 17-38-43](https://github.com/Edgahkipkemoi/mobile/assets/66476329/3b8a6a2c-8e63-4c42-9642-4f73a09f3f5e)

</td>
</tr>
</table>